### ISSUE TITLE
Add App to f-droid

### ISSUE THREAD

**Issue by doronbehar** [y-polek/TransmissionRemote#68]
Hello,
I enjoy using the app but I don't have Google play services neither Google Play store installed on my device. I would have been very relieved if the app was available at [f-droid.org](https://f-droid.org) so I would be connected to updates.
Thanks.

**Comment by zero77** [comment]
Adding apps to f-droid tutorial.
https://f-droid.org/en/tutorials/create-repo/

**Comment by nicd23** [comment]
F-droid version or a link to download apk file would be great, many of use don't use Play Store.

**Comment by y-polek** [comment]
Yes, Fabric dependency is required only for crash reporting, which is optional.

I'll try to add product [flavor](https://developer.android.com/studio/build/build-variants#product-flavors) without it.
If it not gonna work we will, probably, have to add a separate branch without Fabric dependency.

**Comment by ghost** [comment]
Is release now possible after the removal of the crash analytics service?

**Comment by Poussinou** [comment]
@IzzySoft what about the fabric api key reference in the gradle?

**Comment by IzzySoft** [comment]
@Poussinou `prebuild: sed '/xxx/d'`? Of course, ideally it should not be in the global part nor in the foss flavor. If it's just referenced from a flavor not specified in our build recipe, it should be simply ignored anyway.
